src/relayer/Relayer.t.sol

```diff
@@ -30,6 +33,7 @@ contract TestCollybus is ICollybus {
 }
 
 contract RelayerTest is DSTest {
+    CheatCodes internal cheatCodes = CheatCodes(HEVM_ADDRESS);
     Hevm internal hevm = Hevm(DSTest.HEVM_ADDRESS);
```

Comment:
`CheatCodes` and `Hevm` have the same purpose. `CheatCodes` is superior since it contains `Hevm`'s functionality and additional functionality.

Keep only one of them.